### PR TITLE
added need_slb attribute for swarm

### PR DESCRIPTION
**Pull request by arafato**

### CODE REVIEW

File: alicloud/resource_alicloud_cs_swarm.go [hunk 1]
```diff
@@ -96,6 +96,11 @@ func resourceAlicloudCSSwarm() *schema.Resource {
 					return old != ""
 				},
 			},
+			"need_slb": &schema.Schema{
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_cs_swarm.go]: Please output this parameter in the read method

This parameter can not be modified and please set it to forcenew

Comment (arafato) [reply alicloud/resource_alicloud_cs_swarm.go]: Is this parameter really needed in the read method? If `need_slb` was set to false then slb_id will be empty. So this seems kind of redundant to me?

**Comment by arafato** [comment]
```
go test -run TestAccAlicloudCSSwarm_vpc -v -timeout 1800s                                                                                                                                                                                              
=== RUN   TestAccAlicloudCSSwarm_vpc
--- PASS: TestAccAlicloudCSSwarm_vpc (560.11s)
PASS
ok  	github.com/alibaba/terraform-provider/alicloud	560.167s

go test -run TestAccAlicloudCSSwarm_update -v -timeout 1800s                                                                                                                                                                                         
=== RUN   TestAccAlicloudCSSwarm_update
--- PASS: TestAccAlicloudCSSwarm_update (857.73s)
PASS
ok  	github.com/alibaba/terraform-provider/alicloud	857.787s
```

**Comment by arafato** [comment]
```
go test -run TestAccAlicloudCSSwarm_vpc_noslb -v -timeout 1800s                  
=== RUN   TestAccAlicloudCSSwarm_vpc_noslb
--- PASS: TestAccAlicloudCSSwarm_vpc_noslb (638.27s)
PASS
ok  	github.com/alibaba/terraform-provider/alicloud	638.351s
```

**Comment by arafato** [comment]
```
go test -run TestAccAlicloudCSSwarm_importBasicNoSlb -v -timeout 1800s  
=== RUN   TestAccAlicloudCSSwarm_importBasicNoSlb
--- PASS: TestAccAlicloudCSSwarm_importBasicNoSlb (643.95s)
PASS
ok  	github.com/alibaba/terraform-provider/alicloud	644.009s
```

**Comment by arafato** [comment]
```
=== RUN   TestAccAlicloudCSSwarm_importBasic
--- PASS: TestAccAlicloudCSSwarm_importBasic (774.94s)
=== RUN   TestAccAlicloudCSSwarm_importBasicNoSlb
--- PASS: TestAccAlicloudCSSwarm_importBasicNoSlb (785.02s)
=== RUN   TestAccAlicloudCSSwarm_vpc
--- PASS: TestAccAlicloudCSSwarm_vpc (712.82s)
=== RUN   TestAccAlicloudCSSwarm_vpc_noslb
--- PASS: TestAccAlicloudCSSwarm_vpc_noslb (764.83s)
=== RUN   TestAccAlicloudCSSwarm_update
--- PASS: TestAccAlicloudCSSwarm_update (993.29s)
```

**Pull request closed without merging**